ergani/client.py

```diff
@@ -66,17 +98,52 @@ def _handle_response(
             error_message = extract_error_message(response)
             raise APIError(message=error_message, response=response, payload=payload)
 
+    def _extract_submission_result(
+        self, response: Optional[Response]
+    ) -> List[SubmissionResponse]:
+        """
+        Extracts the submission result from the Ergani API response
+
+        Args:
+            response (Response): The response object from the Ergani API
+
+        Returns:
+            List[SubmissionResponse]: A list of submission responses parsed from the API response
+
+        Raises:
+            ValueError: If the response cannot be parsed into submission responses, indicating an unexpected format
+        """
+
+        if not response:
+            return []
+
+        data = response.json()
+        submissions = []
+
+        for submission in data:
+            submission_date_str = submission["submitDate"]
+            submission_date = datetime.strptime(submission_date_str, "%d/%m/%Y %H:%M")
```

Comment:
Decided it would be best if parsed the string here and returned a proper datetime

Reply:
Agreed.